bundles/org.openhab.binding.metofficedatahub/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,197 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="metofficedatahub"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="bridge">
+
+		<label>MetOffice DataHub Bridge</label>
+		<description>MetOffice DataHub API Bridge</description>
+
+		<properties>
+			<property name="Site Specific API Call Count"/>
+		</properties>
+
+		<config-description>
+			<parameter name="siteApiKey" type="text" required="true">
+				<context>password</context>
+				<label>Site Specific API Key</label>
+				<description>The API Key for the Site Specific subscription in your MET Office Data Hub account</description>
+			</parameter>
+			<parameter name="siteRateDailyLimit" type="integer" unit="s" min="1">
+				<label>API Daily Limit</label>
+				<description>A limit to the number of daily site specific API requests</description>
+				<default>250</default>
+				<advanced>true</advanced>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+	<thing-type id="site">
+
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge"/>
+		</supported-bridge-type-refs>
+
+		<label>Forecast Data</label>
+		<description>Site Specific forecast data</description>
+
+		<channel-groups>
+			<channel-group id="common-data" typeId="site-common-data-grp">
```

Comment:
This group seems to be undocumented in README.md

Reply:
I actually tried to make a channels sibling of the group but it wouldn't let me. I guess it presents nicely anyway in the UI. I have added a brief few bits about this in the readme, which hopefully explain it.

Reply:
Sorry, the addition to README.md is somewhat limited. I don't see how this is usefull yet, so maybe you can show me

Reply:
Yeah I was just trying to find out what this "location" response represents, whether its the sampling data location they base the data on, or just a lookup for the lat/long. Either way I figured actually if I want weather data I likely don't really care where the station is, just the data it's calculating for me. So have removed, as I can't find a good description of what that parameter really is. I figured it can always be added later, if someone asks for it, or its clarified what it is.